selfservice/strategy/webauthn/settings.go

```diff
@@ -356,12 +356,12 @@ func (s *Strategy) PopulateSettingsMethod(r *http.Request, id *identity.Identity
 		}
 	}
 
-	web, err := s.newWebAuthn(r.Context())
+	web, err := webauthn.New(s.d.Config().WebAuthnConfig(r.Context()))
 	if err != nil {
 		return err
```

Comment:
Since `webauthn.New` probably doesn't return errors with stack traces, it would be difficult to find the error if it happened here. So please make sure to wrap them:

```
return errors.WithStack(err)
```

Reply:
Sure, makes sense. Done.